### PR TITLE
Improved Emulicious configuration

### PR DESCRIPTION
**Pull request by Calindro**
- Use integer scaling
- Disable color correction
- Fixed download link

### CODE REVIEW

File: emulators/emulicious.py [hunk 1]
```diff
@@ -10,11 +10,8 @@ def __init__(self):
         super().__init__("Emulicious", "https://emulicious.net/", startup_time=1.0)
     
     def setup(self):
-        res = requests.get("https://emulicious.net/downloads/")
-        url = re.search("data-downloadurl=\"([^\"]+)\"", res.text).group(1)
-        download(url, "downloads/Emulicious.zip")
+        download("https://emulicious.net/download/emulicious/?wpdmdl=205", "downloads/Emulicious.zip")
```
Comment (daid) [inline emulators/emulicious.py:13]: Why this change?

Comment (Calindro) [reply emulators/emulicious.py:13]: It's just directly using the download link instead of searching it. The change is not really necessary. I can remove it from the PR if you want. :)

Comment (daid) [reply emulators/emulicious.py:13]: (me slow, yes, sorry)

I initially had a direct link, and that broke after a while, which is why I opted for the searching for the correct link on the page.

Comment (Calindro) [reply emulators/emulicious.py:13]: > (me slow, yes, sorry)

All good! Time is precious so thanks for taking your time :)

It's odd that the link broke. Maybe my server acted up or something...
So would you prefer me to undo the change of the link?

Comment (daid) [reply emulators/emulicious.py:13]: If you say the link will stay valid, I'm fine with the change.
